messages/en.json

```diff
@@ -2,106 +2,202 @@
+        "\u0002{0}\u0002 is forced to walk under a ladder.",
+        "A villager's black cat walks in front of \u0002{0}\u0002.",
+        "A mad scientist's chemicals are thrown onto \u0002{0}\u0002. When the smoke clears, it is apparent that they are now dead.",
+        "Before their execution, \u0002{0}\u0002 spills some salt. The ensuing bad luck costs them their life.",
+        "Before their execution, \u0002{0}\u0002 accidentally breaks a mirror. The unfortunate soul immediately falls over, dead.",
         "Despite protests, the mob drags their victim to the hanging tree. \u0002{0}\u0002 succumbs to the will of the horde, and is hanged.",
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows.",
         "Before the rope is pulled, \u0002{0}\u0002 pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
         "Before the rope is pulled, \u0002{0}\u0002 throws a grenade at the mob. The grenade explodes early.",
+        "Before the rope is pulled, \u0002{0}\u0002 shouts to the crowd. \"Stop! I'm really a ...\", but they die before finishing their thought.",
+        "Before the rope is pulled, \u0002{0}\u0002 shouts angrily at the crowd.",
+        "The villagers drag \u0002{0}\u0002 to the edge of a cliff. Before they can protest, they slip and fall into the abyss.",
         "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered with the leftover remains of their late comrade.",
+        "The villagers angrily behead \u0002{0}\u0002.",
+        "\u0002{0}\u0002 is thrown to a pack of hungry animals. They doesn't last long.",
+        "The village decides to feed \u0002{0}\u0002 to a den of lions. Soon nothing but bones are left.",
         "For SCIENCE, the villagers throw \u0002{0}\u0002 into a volcano. They discover that \u0002{0}\u0002's melting point is less than that of lava.",
-        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that this doesn't burn that well, after all."
+        "For SCIENCE, the villagers electrocute \u0002{0}\u0002. They discover that \u0002{0}\u0002's body fried quite well.",
+        "For SCIENCE, the villagers give \u0002{0}\u0002 an acid bath. They discover that \u0002{0}\u0002's body was too weak to avoid decomposition.",
+        "The villagers tie \u0002{0}\u0002 to a bunch of balloons. They watch as \u0002{0}\u0002 floats off into the sunset.",
+        "The village decides to exile \u0002{0}\u0002. After they pack up and leave, nobody ever sees them again.",
+        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that this doesn't burn that well, after all.",
+        "The angry villagers tie \u0002{0}\u0002 to a pole and burn them at the stake. It seems that this doesn't burn that well, after all."
     ],
```

Comment:
Don't forget to update the noreveal versions of the messages (follow above reviews).